src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1934,7 +1859,6 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                     var property = sourceMethod.AssociatedSymbol as SourcePropertySymbolBase;
                     if (property is not null && property.IsAutoPropertyWithGetAccessor)
                     {
-                        // https://github.com/dotnet/roslyn/issues/67104: Unify with GenerateMethodBody/SynthesizesLoweredBoundBody
                         return MethodBodySynthesizer.ConstructAutoPropertyAccessorBody(sourceMethod);
```

Comment:
This wasn't unified after all?

Reply:
> This wasn't unified after all?

No. It wasn't. These are not really synthesized methods, they have corresponding syntax and we do not produce fully lowered code for them here. This body has to go through other rewrite phases in order to be able to successfully emit IL and PDB information for it.